### ISSUE TITLE
Package for MELPA

### ISSUE THREAD

**Issue by alphapapa** [alphapapa/org-protocol-capture-html#4]

**Comment by stardiviner** [comment]
I created a recipe on MELPA with a PR at here https://github.com/melpa/melpa/pull/3764

**Comment by alphapapa** [comment]
Thanks but I'm not ready to release it on MELPA yet.  Since it requires several steps of manual configuration outside of Emacs, I want to figure out the best way to present the information for MELPA users first.

**Comment by stardiviner** [comment]
I think it can be put in README temporarily.

**Comment by alphapapa** [comment]
I also need to update the commentary, package info, etc. before putting it on MELPA.

**Comment by stardiviner** [comment]
It's ok. MELPA will not merge this recipe until it is ok.

**Comment by alphapapa** [comment]
Yes, but I'd rather not send them a PR until it's actually ready to be pulled.  Steve is very busy with the PRs that are ready, and they don't need ones that aren't ready cluttering their list.

**Comment by stardiviner** [comment]
OK. I will close that PR.

**Comment by alphapapa** [comment]
Thanks.  I appreciate your help.  I'll try to get it on MELPA soon.

**Comment by stardiviner** [comment]
That will be great.

**Comment by brabalan** [comment]
Are there still plans to get this on MELPA?

**Comment by alphapapa** [comment]
Yeah, I still plan to, but since the package requires manual configuration outside of Emacs and is updated infrequently, it's not a high priority for me at the moment.  :)

**Comment by ragloo** [comment]
Great package, many thanks !
It is so great that it should be pulled to MELPA :)

**Comment by alphapapa** [comment]
@ragloo Thanks, glad you're finding it useful.  I'll probably get to this soon when I finish up a few things.

**Comment by non-Jedi** [comment]
Looks like most of the required manual config on this package is strictly to get
vanilla `org-protocol` working. `org-protocol` is now included in
org-plus-contrib (don't know how long it's been included), which isn't on melpa,
but it is an elpa package. Seems to me if figuring out how to include
documentation for getting started with `org-protocol` is what's keeping you from
publishing to melpa, might be org-mode's problem, not yours. ;)

**Comment by non-Jedi** [comment]
Looks like most of the required manual config on this package is strictly to get
vanilla `org-protocol` working. `org-protocol` is now included in
org-plus-contrib (don't know how long it's been included), which isn't on melpa,
but it is an elpa package on orgmode.org. Seems to me if figuring out how to
include documentation for getting started with `org-protocol` is what's keeping
you from publishing to melpa, might be org-mode's problem, not yours. ;)

**Comment by stardiviner** [comment]
I don't know shell much, I think the main problem might on the `org-protocol-capture-html.sh` shell script. If it's possible to use elisp in Emacs. Then to be a package is ok. Also Emacs support shell in some way.

**Comment by Thaodan** [comment]
> Looks like most of the required manual config on this package is strictly to get vanilla `org-protocol` working. `org-protocol` is now included in org-plus-contrib (don't know how long it's been included), which isn't on melpa, but it is an elpa package on orgmode.org. Seems to me if figuring out how to include documentation for getting started with `org-protocol` is what's keeping you from publishing to melpa, might be org-mode's problem, not yours. ;)

The org-protocol is now included in the desktop file for  Emacs 29, meaning that for BSD and Linux there's no extra setup needed besides the one already decsribed in the org-mode manual to make the protocol work  in each browser.

**Comment by alphapapa** [comment]
@Thaodan Thanks, that's good news.
